source/index.ts

```diff
@@ -52,21 +53,22 @@ const defaults: Defaults = {
 			beforeError: [],
 			afterResponse: []
 		},
+		cache: undefined,
```

Comment:
`false` -> `undefined`. Objects should either objects or undefined. People can do `instance.extend({cache: false})` to disable caching later (if the instance has them enabled).